packages/react/src/components/Sheet.tsx

```diff
@@ -84,8 +78,14 @@ export const Sheet = (
             workbookId,
           })
 
-          // this assumes we are only allowing 1 sheet here (which we've talked about doing initially)
-          const sheet = new SheetHandler(workbookSheets[0].id)
+          const thisSheetId = workbookSheets.find((s) => s.slug === config.slug)
+
+          if (!thisSheetId) {
+            throw new Error(
+              `Failed to find sheet slug:${config.slug} in the workbook id: ${workbookId}`
+            )
+          }
+          const sheet = new SheetHandler(thisSheetId.id)
```

Comment:
Since `thisSheetId` is really a sheet.
```suggestion
          const thisSheet = workbookSheets.find((s) => s.slug === config.slug)

          if (!thisSheet) {
            throw new Error(
              `Failed to find sheet slug:${config.slug} in the workbook id: ${workbookId}`
            )
          }
          const sheet = new SheetHandler(thisSheet.id)
```